canonical-web-and-design/cn.ubuntu.com
# cms desktop template

### Done

Add a cms-block template to aid the styling of CMS-included content
Also added CMS'd desktop section as an example of its use. 
Cleaned up old JS that interferes with links.
### QA

make import-new-pages

visit /desktop and ensure that the dynamic content is there and looks good. Some style choices have been made diverging from the live design.
